addons/bus/static/tests/tours/lazy_frontend_start_tour.js

```diff
@@ -0,0 +1,22 @@
+/* @odoo-module */
+
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add("bus.lazy_frontend_start", {
+    test: true,
+    url: "/",
+    steps: () => [
+        {
+            trigger: "body",
+            async run() {
+                // Give 5 seconds before the next step to ensure the bus
+                // would be started if start was called.
+                await new Promise((res) => setTimeout(res, 5000));
```

Comment:
Can't we find a better way to detect it? 5s is a lot of time to wait (and not really guaranteed to work).

Reply:
I can't think of something else since we're waiting for something *not* to happen^^  In theory, the web client is already started when the tour starts to the `bus.start` method would already have been called. I think adding a bit of delay is best to make sure we don't miss anything. What about reducing the delay to 2s or so?